reference/resources/apple_pay_certificate.yaml

```
@@ -0,0 +1,84 @@
+---
+title: Apple Pay Certificate
+type: object
+description: |-
+  An Apple Pay certificate record.
+x-tags:
+  - Apple Pay Certificates
+
+properties:
+  type:
+    type: string
+    description: The type of this resource. Is always `apple-pay-certificate`.
+    example: apple-pay-certificate
+    enum:
+      - apple-pay-certificate
+
+  id:
+    type: string
+    example: fe26475d-ec3e-4884-9553-f7356683f7f9
+    description: The unique identifier for this Apple Pay certificate record.
+    format: uuid
+
+  status:
+    type: string
+    description: The status of this Apple Pay certificate.
+    example: enabled
+    enum:
+      - enabled
```

```suggestion
      - active
```

Let's change it as agreed

---

Changed: https://github.com/gr4vy/gr4vy-openapi/pull/253/commits/e697131dd7235e06b35a05e2b3674dd57a6b801c